torchci/lib/bot/cliParser.ts

```diff
@@ -44,9 +44,10 @@ mergeOption.add_argument("-g", "--green", {
   help: "Merge when *all* status checks pass.",
 });
 mergeOption.add_argument("-f", "--force", {
+  metavar: "MESSAGE",
   help:
     "Merge without checking anything. This requires a reason for auditting purpose, for example:\n" +
-    "`@pytorchbot merge -f '[MINOR] Fix lint. Expecting all PR tests to pass'`\n" +
+    "`@pytorchbot merge -f 'Minor update to fix lint. Expecting all PR tests to pass'`\n" +
     "The reason must be longer than 2 words. ONLY USE THIS FOR CRITICAL FAILURES.",
```

Comment:
nit: some of the examples we listed isn't only for critical failures. Maybe we can delete this line.

Reply:
Yeah, you're right. It's too early to enforce this to only critical issues. At least from what I see in the audit.